assets/circuit_latex_render.mak

```diff
@@ -6,7 +6,9 @@
 
 \begin{document}
 
-\begin{tikzpicture}
+\hoffset=-1in
+\voffset=-1in
+\setbox0\hbox{\begin{tikzpicture}
     \tikzstyle{paulicomponent} = [draw,draw=none,fill=white,minimum size=1.5em] 
     \tikzstyle{phase} = [draw,fill,shape=circle,minimum size=5pt,inner sep=0pt]
     \matrix[row sep=0.4cm, column sep=0.4cm] (circuit) {
```

Comment:
Just below here at line 16=>18 
`\node (q${i}) {$q_1$}; &[-0.1cm]` I think you meant `\node (q${i}) {$q_`**${i}**`$}; &[-0.1cm]`

Reply:
Can't make a suggestion because it's not in the diff

Reply:
good catch